OWML.ModHelper.Menus/ModMenuWithSelectables.cs

```diff
@@ -15,21 +14,23 @@ public class ModMenuWithSelectables : ModPopupMenu, IModMenuWithSelectables
         public event Action OnCancel;
 
         protected List<Selectable> Selectables;
-        protected ModCommandListener CommandListener;
 
-        protected ModMenuWithSelectables(IModConsole console) : base(console) { }
+        private readonly IModEvents _events;
+
+        protected ModMenuWithSelectables(IModConsole console, IModEvents events) : base(console)
+        {
+            _events = events;
+        }
 
         private void SetupCommands()
         {
-            var listenerObject = new GameObject("ConfigurationMenu_Listener");
-            CommandListener = listenerObject.AddComponent<ModCommandListener>();
-            CommandListener.AddToListener(InputLibrary.confirm);
-            CommandListener.AddToListener(InputLibrary.enter2);//keypad's Enter
-            CommandListener.AddToListener(InputLibrary.cancel);
-            CommandListener.AddToListener(InputLibrary.escape);
-            CommandListener.AddToListener(InputLibrary.setDefaults);
-            CommandListener.OnNewlyReleased += OnButton;
-            listenerObject.SetActive(false);
+            _events.Input.AddToListener(InputLibrary.confirm);
+            _events.Input.AddToListener(InputLibrary.enter2);//keypad's Enter
+            _events.Input.AddToListener(InputLibrary.cancel);
+            _events.Input.AddToListener(InputLibrary.escape);
+            _events.Input.AddToListener(InputLibrary.setDefaults);
+            _events.Input.OnNewlyReleased += OnButton;
+            //listenerObject.SetActive(false); todo hmmm? :)
```

Comment:
@TAImatem what should we do here?

Reply:
just change how OnNewlyReleased works I suppose